code/cutscene/ogg/OggDecoder.cpp

```diff
@@ -233,7 +234,7 @@ namespace cutscene
                 vorbis_block_init(&movie.vstate, &movie.vblock);
             }
 
-            auto fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
+            fps = static_cast<float>(movie.tinfo.fps_numerator) / movie.tinfo.fps_denominator;
```

Comment:
Your "goto Error;" skips this declaration. Clang (and gcc) abort with an error because of this.

There's no particular reason why I picked the beginning of the function, so feel free to move the declaration anywhere before the first use of goto.